streamer/src/recvmmsg.rs

```diff
@@ -44,9 +44,9 @@ pub fn recv_mmsg(sock: &UdpSocket, packets: &mut [Packet]) -> io::Result<(usize,
     use std::os::unix::io::AsRawFd;
 
     let mut hdrs: [mmsghdr; NUM_RCVMMSGS] = unsafe { mem::zeroed() };
```

Comment:
iirc, this indeed needed to be zeroed.

Reply:
You'd need to set most of the fields if you don't zero the header so it's probably not worth trying to optimize this.